infrastructure/main.tf

```
@@ -137,38 +120,17 @@ resource "azurerm_key_vault_secret" "POSTGRES_HOST" {
   key_vault_id = data.azurerm_key_vault.default.id
 }
 
+# These two are not exported by the v14 module, but used by the pods
 resource "azurerm_key_vault_secret" "POSTGRES_PORT" {
   count        = local.instance_count
   name         = "${local.default_name}-POSTGRES-PORT"
-  value        = module.idam-testing-support-api-db[0].postgresql_listen_port
+  value        = "5432"
   key_vault_id = data.azurerm_key_vault.default.id
 }
 
 resource "azurerm_key_vault_secret" "POSTGRES_DATABASE" {
   count        = local.instance_count
   name         = "${local.default_name}-POSTGRES-DATABASE"
-  value        = module.idam-testing-support-api-db[0].postgresql_database
-  key_vault_id = data.azurerm_key_vault.default.id
-}
-
-# TEMP for v14
-resource "azurerm_key_vault_secret" "POSTGRES-USER_V14" {
-  count        = local.instance_count
-  name         = "${local.default_name}-POSTGRES-USER-v14"
-  value        = module.idam-testing-support-api-db-v14[0].username
-  key_vault_id = data.azurerm_key_vault.default.id
-}
-
-resource "azurerm_key_vault_secret" "POSTGRES-PASS_V14" {
-  count        = local.instance_count
-  name         = "${local.default_name}-POSTGRES-PASS-v14"
-  value        = module.idam-testing-support-api-db-v14[0].password
-  key_vault_id = data.azurerm_key_vault.default.id
-}
-
-resource "azurerm_key_vault_secret" "POSTGRES_HOST_V14" {
-  count        = local.instance_count
-  name         = "${local.default_name}-POSTGRES-HOST-v14"
-  value        = module.idam-testing-support-api-db-v14[0].fqdn
+  value        = "idamtstsptapi"
```

Why use a hardcoded value?

---

I guess we can  do

```suggestion
  value        = idam-testing-support-api-db-v14[0].database_name
```

just so it gets it from the module where we've already set it.